dallinger/recruiters.py

```diff
@@ -145,6 +154,26 @@ def recruit_participants(self, n=1):
             duration_hours=self.config.get('duration')
         )
 
+    def notify_recruited(self, participant, experiment):
+        """Assign a Qualification to the Participant for the experiment ID,
+        and for the configured group_name, if it's been set.
+        """
+        score = '1'  # always '1'; doesn't matter
```

Comment:
I have the MTurkService code working now, since this was the easiest way to answer the "is it possible" question. What remains is hashing out how to describe these limits in the syntax of the config file (assuming that's where we want to do it).

I think we need to represent 3 related things:

1. Qualification name
2. Comparator (ie: "max", "min", "does not exist" etc., ) For what we've discussed this could always be "max" I think. If you wanted only workers with no experience, you could use "max" with a value of 0.
3. value (what number to use for max/min)